### DISCUSSION TITLE
Ordering at JLCPCB. - "BT1 designator don't exist in the CPL file."

### DISCUSSION THREAD

**Discussion by ErikDB87** [rbaron/b-parasite#212] [Q&A]
![image](https://github.com/user-attachments/assets/6824f80b-a555-48d6-a72a-7b40b37724f1)

I dare to assume this is because I requested only the "bottom side" to be assembled, but I wanted to check with people who know more of it than I.

Should I just delete the 2nd line in the BOM file, and leave all the rest the same? Or is that a bad idea and would there then not be a possible spot to solder the battery clip onto?

Maybe I should just click 'continue'? 🤔

Thanks in advance for your input!

**Comment by rbaron** [comment]
@ErikDB87 this is happening because the files in the repo are for the top side assembly, that's the side with all the components. The bottom has only the battery clip (identified by the `BT1` designator you see there), and that's not usually assembled. Tl;dr: select the top layer assembly.

**Reply by ErikDB87** [reply]
But I did select only the top layer assembly.

I found the file here: https://github.com/rbaron/b-parasite/wiki/How-to-order:-PCB-fabrication-and-SMT-assembly#2-the-pick-and-place-file, which leads me to the file (https://github.com/rbaron/b-parasite/blob/main/kicad/parasite.csv). In this file, I see BT1.

**Reply by rbaron** [reply]
Ah sorry, my mistake. I got confused because in the original post I understood you requested only the _bottom side_ assembly. But I get now you requested the top side. So BT1 (the battery clip) indeed should not be assembled. You should be able to just ignore that message.
